packages/report-server/src/reportBuilder/transform/functions/insertColumns.ts

```diff
@@ -51,7 +51,14 @@ const insertColumns = (table: TransformTable, params: InsertColumnsParams, conte
     values,
   }));
 
-  return table.upsertColumns(columnUpserts);
+  // Drop, then re-insert the original skipped rows
+  const rowsToDrop = Object.keys(skippedRows).map(rowIndexString => parseInt(rowIndexString));
```

Comment:
```suggestion
  const rowsToDrop = Object.keys(skippedRows);
```
Should keys of skippedRows are number?

Reply:
Unfortunately even though we set them as numbers, they come out as strings. That's JS for you 🤷